clang/lib/CodeGen/CodeGenFunction.cpp

```diff
@@ -892,6 +892,10 @@ void CodeGenFunction::StartFunction(GlobalDecl GD, QualType RetTy,
     }
   }
 
+  if (FD->hasAttr<NakedAttr>()) {
```

Comment:
Is this change needed given the separate change in LLVM skipPGOGen?

Reply:
This should not be needed. The llvm change is sufficient to work with all frontends including clang.

Reply:
not at all, that was a leftover, sorry about that.